### PR TITLE
PT-BR Translation pt.po

### PR DESCRIPTION
**Pull request by mauricio-sonegatti**
Creation of pt-br translation, based on @joebordes translation for spanish.
The file had to be changed to pt.br because the pt-br.po file does not build.

### CODE REVIEW

**Comment by mazameli** [comment]
Wow! This is great, Mauricio. Just so you know, a few minutes ago we merged https://github.com/metabase/metabase/pull/7372 into master which externalizes a number of backend strings, plus a handful of front-end ones we missed.

File: locales/pt.po [hunk 1]
```diff
@@ -0,0 +1,8573 @@
...
+"rescan it periodically to keep the metadata up-to-date. You can control when "
+"the periodic rescans happen below."
+msgstr ""
+"Para fazer um pouco de sua mágica, o Metabase precisa varrer o banco de "
+"dados. Faremos isso periodicamente para manter os metadados atualizados. "
+"Você pode controlar quando as revisões periódicas são feitas a seguir."
+
+#: frontend/src/metabase/admin/databases/components/DatabaseSchedulingForm.jsx:125
+msgid "Database syncing"
+msgstr "Sincronizando Banco de Dados"
```
Comment (fernandosjp) [inline locales/pt.po]: I would keep database in lower case "banco de dados"

**Review by fernandosjp — COMMENTED** [review]
@mauricio-sonegatti I didn't have time to read the full document but I noted some stuff would be worth looking at! If you need help let us know here!

File: locales/pt.po [hunk 2]
```diff
@@ -0,0 +1,8887 @@
...
+msgid "Add Database"
+msgstr "Adicionar banco de dados"
+
+#: frontend/src/metabase/admin/databases/containers/DatabaseEditApp.jsx:150
+msgid "Connection"
+msgstr "Conexão"
+
+#: frontend/src/metabase/admin/databases/containers/DatabaseEditApp.jsx:150
+msgid "Scheduling"
+msgstr "Programação"
```
Comment (dannyeuu) [inline locales/pt.po]: I think @mauricio-sonegatti should be a better translation `Agendamento`

File: locales/pt.po [hunk 3]
```diff
@@ -0,0 +1,8887 @@
...
+#: frontend/src/metabase/admin/datamodel/containers/SegmentForm.jsx:84
+#: frontend/src/metabase/admin/people/components/EditUserForm.jsx:237
+#: frontend/src/metabase/admin/settings/components/SettingsEmailForm.jsx:249
+#: frontend/src/metabase/admin/settings/components/SettingsLdapForm.jsx:258
+#: frontend/src/metabase/admin/settings/components/SettingsSlackForm.jsx:217
+#: frontend/src/metabase/pulse/components/PulseEdit.jsx:194
+#: frontend/src/metabase/query_builder/components/AlertModals.jsx:336
+#: frontend/src/metabase/reference/components/RevisionMessageModal.jsx:47
+msgid "Save changes"
+msgstr "Salvar mudanças"
```
Comment (dannyeuu) [inline locales/pt.po]: Here `mudanças` to `alterações`

File: locales/pt.po [hunk 4]
```diff
@@ -0,0 +1,8887 @@
...
+msgstr "Sincronize o esquema do banco de dados agora"
+
+#: frontend/src/metabase/admin/databases/containers/DatabaseEditApp.jsx:206
+#: frontend/src/metabase/admin/databases/containers/DatabaseEditApp.jsx:218
+#: frontend/src/metabase/admin/datamodel/containers/FieldApp.jsx:783
+#: frontend/src/metabase/admin/datamodel/containers/FieldApp.jsx:791
+#: frontend/src/metabase/admin/datamodel/containers/TableSettingsApp.jsx:112
+#: frontend/src/metabase/admin/datamodel/containers/TableSettingsApp.jsx:120
+msgid "Starting…"
+msgstr "Começando ..."
```
Comment (dannyeuu) [inline locales/pt.po]: here `Iniciando`

File: locales/pt.po [hunk 5]
```diff
@@ -0,0 +1,8887 @@
...
+#: frontend/src/metabase/parameters/components/ParameterWidget.jsx:173
+#: frontend/src/metabase/pulse/components/PulseListItem.jsx:55
+#: frontend/src/metabase/query_builder/components/AlertListPopoverContent.jsx:205
+#: frontend/src/metabase/reference/components/EditButton.jsx:18
+msgid "Edit"
+msgstr "Editar"
+
+#: frontend/src/metabase/admin/datamodel/components/ObjectActionSelect.jsx:59
+msgid "Revision History"
+msgstr "Rever História"
```
Comment (dannyeuu) [inline locales/pt.po]: Maybe can be better with `Rever Histórico`

File: locales/pt.po [hunk 6]
```diff
@@ -0,0 +1,8887 @@
...
+msgstr "Falhou"
+
+#: frontend/src/metabase/admin/datamodel/components/ObjectRetireModal.jsx:61
+msgid "Success"
+msgstr "Sucesso"
+
+#: frontend/src/metabase/admin/datamodel/components/PartialQueryBuilder.jsx:118
+#: frontend/src/metabase/public/components/widgets/AdvancedEmbedPane.jsx:110
+msgid "Preview"
+msgstr "Vista preliminar"
```
Comment (dannyeuu) [inline locales/pt.po]: Maybe `Visualizar`

**Review by dannyeuu — COMMENTED** [review]
It should be reviewed this translation concordance in the Portuguese language, looks like translated by Google Translator.

**Comment by mauricio-sonegatti** [comment]
As stated on the 1a01c68 commit... I updated the translations, some weird behavior happened in update-translations. Something that wasn't marked fuzzy might escaped me.
For now the translation seems adequate, but I can't test it.
After building the latest version, the translation isn't showing. Even selecting it with the admin option it doesn't do anything. 
I'm gonna keep trying.

**Comment by kdoh** [comment]
Going to go ahead and close this to reflect our switch to POEditor for translations. [Here's the PR-BR page](https://poeditor.com/projects/po_edit?id_language=190&id=200535) there in case anyone needs it.

**Pull request closed without merging**